compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUOps.td

```diff
@@ -289,66 +296,89 @@ def IREEGPU_CoalescedGatherDMAOp : Op<IREEGPU_Dialect, "coalesced_gather_dma", [
 
     ## Operands and Results
 
-    * `$indices`: Tensor/memref of index type specifying gather locations in
-      the source tensor
-    * `$source`: Source tensor/memref containing the data to be gathered
+    * `$indices`: The variadic `indices` operand is an optional tensor or vector of indices
+      to gather from. If the indices are present, their shape must be a prefix of
+      the `init`/`result` type.
```

Comment:
Each `index` must be a 1D tensor or vector whose length matches the length of the corresponding dimension of `source`